.github/workflows/main.yml

```diff
@@ -134,6 +134,12 @@ jobs:
       - name: check go modules tidiness 
         run: make tidy-module-check
 
+      - name: install protolint
+        run: ./scripts/install_protolint.sh
+
+      - name: lint proto files
+        run: find . -name "*.proto" -print0 | xargs -0 ./protolint lint
```

Comment:
Would be nice if we had a `make` goal for the installation and lint step as well, so developers could run it locally too (instead of always needing to push and let GitHub run the CI).

Reply:
Yes, this makes sense. Thanks!